by_id/openroad.rcx/003-success-disconnected-ios/handler.py

```diff
@@ -1,27 +1,39 @@
+import os
 import subprocess
 
+from openlane.common import get_script_dir
 
-def handle(step):
 
+def handle(step):
     print(step.state_out["spef"])
+
+    env = os.environ.copy()
+    env[
+        "PYTHONPATH"
+    ] = f"{env.get('PYTHONPATH', '')}:{os.path.join(get_script_dir(), 'odbpy')}"
     for corner in ["nom", "min", "max"]:
+        print(f"--- Corner {corner} --- ")
         spef_file = step.state_out["spef"][f"{corner}_*"]
         with open("run_sta.tcl", "w") as f:
-            lib = step.config["LIB"]["*_tt_025C_1v80"][0]
-            f.write(f"read_liberty {lib}\n")
-            f.write(f"read_verilog {step.state_out['nl']}\n")
-            f.write("read_verilog ./aes_example.v\n")
-            f.write("link_design user_project_wrapper\n")
-            f.write(f"read_spef {spef_file}\n")
-            f.write(f"read_spef -path mprj ./aes_example.{corner}.spef\n")
-            f.write("report_parasitic_annotation -report_unannotated > out.log\n")
-
-        result = subprocess.run(["sta", "-exit", "run_sta.tcl"], capture_output=True)
+            f.write(
+                open("run_sta.tcl.tpl")
+                .read()
+                .format(
+                    lib=step.config["LIB"]["*_tt_025C_1v80"][0],
+                    nl=step.state_out["nl"],
+                    spef_file=spef_file,
+                    corner=corner,
+                )
+            )
 
-        if result.stderr:
```

Comment:
This was left for debugging in case of test failure

Reply:
Probably removed it by mistake, will handle

Reply:
Handled